script-with-argparse.py

```diff
@@ -0,0 +1,223 @@
+#! /usr/bin/python
+
+import datetime
+import re
+
+"""This script gives a translation agency an overview of all its projects.
+"""
+import argparse
+
+class Project:
+
+    def __init__(self, title, client, source, target, words, start, deadline, price, tm, translator = 'internal', revisor = 'internal', status = 'created', domain = ''):
```

Comment:
For readability, I would recommend separating the arguments in different lines.

Reply:
Done, thanks for the tip! :)